unity-renderer/Assets/Textures/UI/Common/FavoriteIcon.png.meta

```diff
@@ -6,7 +6,7 @@ TextureImporter:
   serializedVersion: 12
   mipmaps:
     mipMapMode: 0
-    enableMipMap: 1
+    enableMipMap: 0
     sRGBTexture: 1
     linearTexture: 0
```

Comment:
strange that this .meta file remained even that you removed original .png. Maybe this .meta-file is the reason why I have both files when pull you branch
![image](https://github.com/decentraland/unity-renderer/assets/35366872/8dea13f2-7317-4bd3-bfc2-ce7b8378a6dd)